### ISSUE TITLE
Add HTML editor for email body.

### ISSUE THREAD

**Issue by pysaquib** [navgurukul/shayara-bano#5]
Add an HTML editor on the UI so that user is able to format their email content

**Comment by abhishekgupta92** [comment]
I don't think this is a good approach. At some point, we should introduce a
simple HTML editor which are available on Frontend and simplify the user
experience.

I feel - you guys need to think of what users already do, rather than
building complicating dashboards for even simple use cases.

Abhishek Gupta <https://medium.com/@abhishekgupta92>
CEO & Co-founder
Read about us at Forbes India 30u30
<https://www.forbesindia.com/article/30-under-30-2020/navgurukul-cracking-the-code-to-success/57717/1>
, TOI
<https://timesofindia.indiatimes.com/home/sunday-times/these-school-dropouts-have-cracked-the-code-to-a-better-life/articleshow/71458732.cms>
or ZDNet
<http://www.zdnet.com/article/this-home-for-underprivileged-coders-will-put-the-indian-government-and-the-it-industry-to-shame/>
or
follow our blog <http://medium.com/navgurukul> for regular updates.
You can contribute now using navgurukul.org/donate. Each rupee matters.

**Comment by pysaquib** [comment]
@abhishekgupta92 Like option to bold and italicize text in the UI itself?

**Comment by abhishekgupta92** [comment]
Yes. There is no reason that these utilities should be different than Gmail
Similar to the Gmail interface the better.

Abhishek Gupta <https://medium.com/@abhishekgupta92>
CEO & Co-founder
Read about us at Forbes India 30u30
<https://www.forbesindia.com/article/30-under-30-2020/navgurukul-cracking-the-code-to-success/57717/1>
, TOI
<https://timesofindia.indiatimes.com/home/sunday-times/these-school-dropouts-have-cracked-the-code-to-a-better-life/articleshow/71458732.cms>
or ZDNet
<http://www.zdnet.com/article/this-home-for-underprivileged-coders-will-put-the-indian-government-and-the-it-industry-to-shame/>
or
follow our blog <http://medium.com/navgurukul> for regular updates.
You can contribute now using navgurukul.org/donate. Each rupee matters.
